packages/design-system/src/Radio/index.tsx

```diff
@@ -98,6 +101,7 @@ export const Radio = styled.label<{
 
   input:disabled ~ .checkbox:after {
     background-color: var(--ads-radio-disabled-background-color);
+    //border: var(--ads-radio-disabled-border-color);
```

Comment:
?

Reply:
Was trying something out. Will remove this, my bad.